tests/postgres/ttables.nim

```diff
@@ -80,7 +80,7 @@ suite "Table creation":
     ]
 
     check dbConn.getAllRows(qry, "Pet") == @[
-      @[?"favtoy", ?"bigint"],
+      @[?"favToy", ?"bigint"],
```

Comment:
An example for sudden case-sensitivity, though that is when comparing column names and not during actual queries, not sure how much that matters.